guides/common/modules/con_host-substatus-overview.adoc

```diff
@@ -169,76 +169,6 @@ Possible values:
 | SCA enabled | OK | 5
 |===
 
-Service level::
```

Comment:
Subscription status is also removed. Need to remove the section above and all other mentions of subscription status.